app/static/js/calendar-util.js

```diff
@@ -189,3 +134,81 @@ export const updateTimeSlots = (event) => {
     nextDateElement.appendChild(clone);
   }
 };
+
+const renderDateHeaders = (selectedDate) => {
+  /*
+   NOTE: instead of adding the day-type data attribute to the markup, you can
```

Comment:
@pwaller815 if you want to add a commit to this PR with this suggested refactor go ahead. otherwise just resolve the conversation and merge